pyproject.toml

```diff
@@ -112,19 +109,25 @@ line-length = 100
 target-version = ["py38", "py39", "py310", "py311"]
 preview = true
 
-[tool.isort]
-profile = "black"
+[tool.ruff]
```

Comment:
Is there any flake8 config copy within pyproject.toml to configure ruff?

Reply:
I moved all the relevant flake8 configuration to pyproject.toml:
*` max-line-length` is just `line-length`
* moved `max-doc-length = 100` to pycodestyle
* fixed B028 errors
* E203 is missing from [the ruleset ](https://beta.ruff.rs/docs/rules)
* added more folder ignores